docs/cluster_spec.md

```diff
@@ -159,9 +158,39 @@ Will result in the flag `--feature-gates=ExperimentalCriticalPodAnnotation=true,
 
 ### networkID
 
-On AWS, this is the id of the VPC the cluster is created in. If creating a cluster from scratch, this field doesn't need to be specified at create time; `kops` will create a `VPC` for you.
+On AWS, this is the id of the VPC the cluster is created in. If creating a cluster from scratch, this field does not need to be specified at create time; `kops` will create a `VPC` for you.
 
 ```yaml
 spec:
   networkID: vpc-abcdefg1
 ```
+
+More information about re-using a VPC is [here](run_in_existing_vpc.md).
```

Comment:
More information about using `kops` with an existing VPC is here:

When I hear "re-using" a VPC, I think of using a dirty old VPC. I'm strange.